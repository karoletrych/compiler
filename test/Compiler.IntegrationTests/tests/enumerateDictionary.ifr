module EnumerateDictionary

fun main
{
    val dict = new System::Collections::Generic::Dictionary<int,string>();
    dict.Add(1, "one");
    dict.Add(2, "two");
    dict.Add(4, "four");

    val enumerator = dict.GetEnumerator();
    while(enumerator.MoveNext())
    {
        System::Console:.WriteLine(enumerator.Current.ToString());
    }
    enumerator.Dispose();
}

