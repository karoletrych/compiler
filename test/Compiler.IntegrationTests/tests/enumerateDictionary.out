[1, one]
[2, two]
[4, four]
